requirements/doc.txt

```diff
@@ -44,7 +44,7 @@ coverage[toml]==6.4.2
     # via
     #   -r requirements/test.txt
     #   pytest-cov
-doc8==0.11.2
+doc8==1.0.0
```

Comment:
Only major change is drop of support for Python 3.6
https://github.com/PyCQA/doc8/releases/tag/v1.0.0